public/js/dev/game/init.js

```diff
@@ -345,7 +345,7 @@ function drawPlayers() {
             height: constants.HAND_POS[nump][j].h * globals.ui.h,
             rotation: constants.HAND_POS[nump][j].rot,
             align: 'center',
-            reverse: j === 0,
+            reverse: (lobby.reverseHand ? j != 0 : j === 0),
```

Comment:
This isn't the same 'branch' as the other code. I suggest not modifying it at the same time as the other branch.